docs/dev/Pagination-v2.md

```diff
@@ -100,10 +111,381 @@ The discussion below uses *under max_result_window* to refer to scenarios that c
 ## SQL Node Load Balancing
 V2 SQL engine supports *sql node load balancing* -- a cursor request can be routed to any SQL node in a cluster. This is achieved by encoding all data necessary to retrieve the next page in the `cursor_id`.
 
-## Design Diagrams
-New code workflows are highlighted.
+## Feature Design
+
```

Comment:
One thing that is not defined in this section is "How do we determine what is a paged request vs a non-paged request?".  We should say where it's checked and what it checks to make this determination.

Reply:
Fixed in d3888ceaf